rust/lance-core/src/datatypes/field.rs

```diff
@@ -476,13 +432,13 @@ impl Field {
     ///
     /// If the ids are `[2]`, then this will include the parent `0` and the
     /// child `3`.
-    pub(crate) fn project_by_ids(&self, ids: &[i32]) -> Option<Self> {
```

Comment:
Can you add `include_all_children` to the method docs.  This is in `lance-core` so it is used everywhere.